tools/ansible/tasks/wsu/test/e2e/wsu_test.go

```diff
@@ -90,11 +96,16 @@ func TestWSU(t *testing.T) {
 	require.NotEmptyf(t, playbookPath, "WSU_PATH environment variable not set")
 	require.NotEmptyf(t, clusterAddress, "CLUSTER_ADDR environment variable not set")
 
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	require.NoError(t, err, "Could not build kubeconfig")
```

Comment:
This should be `could not build config from flags`.

Nit: You also capitalize your sentences for errors and message which I think is not per go conventions